docs/packages/cookbook/collecting-and-querying-sentiment.rst

```diff
@@ -0,0 +1,91 @@
+Collecting and Querying Sentiment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example package collects plain text files and stores them in Steamship. It runs a sentiment analysis
+package (OneAI) on them as they are added.  Two endpoints allow fetching all of the positive and
+negative tagged documents.
+
+..
+    TODO: Test this once oneai tagger is updated
+
+.. code-block:: python
+
+    """
+    This package accepts text documents, stores them, and runs sentiment analysis.
+    """
+    from typing import Type, Dict, Any
+
+    from steamship import File, Steamship
+    from steamship.data import TagKind, SentimentTag
+    from steamship.invocable import Config, create_handler, post, PackageService
+
+
+    class SimpleSentimentPackage(PackageService):
+
+        def __init__(self, client: Steamship, config: Dict[str, Any] = None):
+            super().__init__(client, config)
+
+            # Instantiate a sentiment plugin.
+            self.sentiment_plugin_instance = client.use_plugin(
+                plugin_handle="oneai-tagger",
+                instance_handle="my-sentiment-analyzer",
+                config={"skills": "sentiment"}
+            )
+
+        # The config method allows your package to return a class
+        # that defines its required configuration.
+        # See <link> for more details.  This package doesn't have any specific
+        # required configuration, so we return the default Config object.
+        def config_cls(self) -> Type[Config]:
```

Comment:
is it possible to make `config_cls` an abstract method on `PackageService` ?  That would simplify these examples, making them easier to digest.

Reply:
I don't know the answer to this.  Python's abstract class stuff is still a little mystifying to me.  @EniasCailliau Thoughts?